### PR TITLE
Changelog 2.0

### PR DESCRIPTION
**Pull request by dennisreimann**

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -1,5 +1,93 @@
...
+* Developers leveraging the Greenfield API should check our breaking changes to ensure 2.0 compatibility
+
+### New feature
+
+* Interface localization (#5662 #6118 #6119 #6120 #6203 #6271 #6292 #6302) @NicolasDorier @dennisreimann
+* New sidebar navigation (#5744 #6267)  @dstrukt @dennisreimann
+* Improved onboarding flow (#6188 #6231 #6233)  @dstrukt @dennisreimann @pavlenex
+* Improved branding options (#5947 #5992) @dennisreimann @dstrukt
+* Support pluginable rate providers (#5777) @Kukks
+* User: Add name and profiole picture (#6008) @dennisreimann @NicolasDorier
```
Comment (ndeet) [inline Changelog.md:19]: typo "profile"

Comment (dennisreimann) [inline Changelog.md:19]:
```suggestion
* User: Add name and profile picture (#6008) @dennisreimann @NicolasDorier
```

File: Changelog.md [hunk 2]
```diff
@@ -1,5 +1,96 @@
...
+* Boltcard: Require non interactivity for payments (#6289) @NicolasDorier
+* LND: Upgrade to 0.18.3-beta (#6303) @rockstardev
+
+## Deprecations
+
+* Remove experimental custodian accounts feature (#5863 #6193) @Kukks
+* Remove Checkout V1 (#5906) @NicolasDorier
+* Remove MySQL and SQlite dependencies (#5910) @NicolasDorier
+* Remove period concept from PullPayment (#5963) @NicolasDorier
+* Remove the Altcoins build (#6177) @NicolasDorier
```
Comment (pavlenex) [inline Changelog.md:91]: @NicolasDorier how does this affect existing users who use this build? Considering altcoins aren't yet moved to plugins, can users updating to 2.0 continue to use them?

Comment (gruve-p) [reply Changelog.md:93]: I would like to know this too. GRS uses this build.

Comment (NicolasDorier) [reply Changelog.md:96]: @gruve-p nothing to do. We remove the build flags, long term goal is to isolate altcoins in different plugins, but that's not done here. It's like if the build now is like the altcoins build.

File: Changelog.md [hunk 3]
```diff
@@ -1,5 +1,101 @@
...
+* Developers leveraging the Greenfield API should check our breaking changes to ensure 2.0 compatibility
+
+### New feature
+
+* Interface localization (#5662 #6118 #6119 #6120 #6203 #6271 #6292 #6302 #6318) @NicolasDorier @dennisreimann
+* New sidebar navigation (#5744 #6267)  @dstrukt @dennisreimann
+* Improved onboarding flow (#6188 #6231 #6233)  @dstrukt @dennisreimann @pavlenex
+* Improved branding options (#5947 #5992) @dennisreimann @dstrukt
+* Support pluginable rate providers (#5777) @Kukks
+* User: Add name and profiole picture (#6008) @dennisreimann @NicolasDorier
```
Comment (Kukks) [inline Changelog.md:19]:
```suggestion
* User: Add name and profile picture (#6008) @dennisreimann @NicolasDorier
```

**Pull request merged**